src/js/player.js

```diff
@@ -2481,23 +2481,14 @@ class Player extends Component {
 }
 
 /*
- * Global player list
- *
- * @type {Object}
- */
-Player.players = {};
```

Comment:
If you make `videojs.options` just an alias to `Player.options_`, it's easy to get them out of sync and be confused by the results. For instance:
```js
videojs.options = {
  techOrder: ['youtube']
}
var player = videojs(video);
```
`player` won't be using the YouTube tech and you'd have to dig around in the source quite a bit to figure out why.

I tried keeping `Player.players` as-is which didn't work initially. I don't recall why off-hand. I'll have to give it a shot now and see if the changes to `options` make is possible.